llvm/lib/DebugInfo/GSYM/GsymCreator.cpp

```diff
@@ -385,6 +385,12 @@ uint32_t GsymCreator::insertString(StringRef S, bool Copy) {
   return StrOff;
 }
 
+StringRef GsymCreator::getString(uint32_t offset) {
```

Comment:
Can we return an optional value like std::optional<StringRef>? The release build ignores asserts. I'd make it explicit for its use.

Reply:
@kyulee-com could you clarify what you meant here ?

Reply:
If you use `StringOffsetMap->at(offset)` then it will abort if the key doesn't exist. However you don't get to pass a custom error message.